src/name_bazaar/ui/events/public_resolver_events.cljs

```diff
@@ -63,9 +59,57 @@
  interceptors
  (fn [{:keys [:db]} [addr name]]
-   (info [:REVERSE-RESOLVED-ADDR addr name])
    (when (and name
               (not= name ""))
      {:db (assoc-in db [:public-resolver/reverse-records
                         addr
                         :public-resolver.record/name] name)})))
+
+(reg-event-fx
+ :public-resolver/set-addr
+ [interceptors (validate-first-arg (s/keys :req [:ens.record/name
+                                                 :ens.record/addr]))]
+ (fn [{:keys [:db]} [form-data]]
+   (let [instance (get-instance db :ens)
+         form-data (assoc form-data
+                          :ens.record/node (namehash (str (:ens.record/name form-data)
+                                                          constants/registrar-root)))]
+     {:dispatch [:district0x/make-transaction
+                 {:name (gstring/format "Point %s to %s"
+                                        (:ens.record/name form-data)
+                                        (:ens.record/addr form-data))
+                  :contract-key :public-resolver
+                  :contract-method :set-addr
+                  :form-data (select-keys form-data [:ens.record/node :ens.record/addr])
+                  :args-order [:ens.record/node :public-resolver]
+                  :result-href (path-for :route.ens-record/detail form-data)
+                  :form-id (select-keys form-data [:ens.record/node])
+                  :tx-opts {:gas 100000 :gas-price default-gas-price}
+                  :on-tx-receipt [:district0x.snackbar/show-message
+                                  (gstring/format "%s is now pointing to %s"
+                                                  (:ens.record/name form-data)
+                                                  (truncate (:ens.record/addr form-data)
+                                                            10))]}]})))
+
+(reg-event-fx
+ :public-resolver/set-name
+  [interceptors (validate-first-arg (s/keys :req [:ens.record/addr]))]
+  (fn [{:keys [:db]} [form-data]]
+    (let [form-data (-> form-data
+                        (update :ens.record/name #(str % constants/registrar-root))
+                        (assoc :ens.record/node (reverse-record-node (:ens.record/addr form-data))))]
+      {:dispatch [:district0x/make-transaction
+                  {:name (gstring/format "Point %s to %s"
+                                         (:ens.record/addr form-data)
+                                         (:ens.record/name form-data))
+                   :contract-key :public-resolver
+                   :contract-method :set-name
+                   :form-data (select-keys form-data [:ens.record/node :ens.record/name])
+                   :args-order [:ens.record/node :ens.record/name]
+                   :form-id (select-keys form-data [:ens.record/node])
+                   :tx-opts {:gas 100000 :gas-price default-gas-price}
+                   :on-tx-receipt [:district0x.snackbar/show-message
```

Comment:
As I said before, when I point address to a name. `:on-tx-receipt` should reload reverse records for `:my-addresses`, so new name is actually displayed in UI, in top left corner.

Reply:
Should reload reverse record only for this `:ens.record/addr`, not for `:my-addresses`, sry